tools/install-powershell-readme.md

```diff
@@ -24,7 +24,7 @@
 * -includeide - installs vscode and vscode powershell extension (only relevant to machines with desktop environment)
 * -interactivetesting - do a quick launch test of vscode - only relevant when used with -includeide
 * -skip-sudo-check - use sudo without verifying it's availability (hard to accurately do on some distros)
-* -allowprerelease - includes prereleases of powershell core when selection allows this.  Repository based installs take the latest on the repo url regardless of this switch.
```

Comment:
technically, this is a breaking change, but not in the product

Reply:
I think it is fine since it hasn't been in for long

Reply:
Yeah, I think it was only working for 24-36 hours.

Reply:
Agreed, this lgtm, but I'm supposed to give 24 hours for review.

Reply:
Also these changes are just trying to keep up with significant "breaking change" design decisions - like establishing the entirely separate package id for previews.  It's a great change - but such shifts are the lions share of "breaking" compared to the breaking code changes needed to support them.